pages.fr/common/git-imerge.md

```diff
@@ -1,29 +1,29 @@
 # git-imerge
 
-> Génerer un merge ou un rebase entre deux branches de maniére incrémentale.
+> Générer un `git merge` ou un `git rebase` entre deux branches de manière incrémentale.
 > Les conflits entre les branches sont suivis en paires de commits individuels, pour simplifier la résolution des conflits.
 > Plus d'informations : <https://github.com/mhagger/git-imerge>.
 
-- Démarrer un i-merge rebase (se placer dans la branche a rebaser d'abord) :
+- Démarrer un imerge rebase (se placer dans la branche à rebaser d'abord) :
 
 `git imerge rebase {{branche_sur_laquelle_rebaser}}`
 
-- Démarrer imerge merge (se placer dans la branche depuis laquelle merger d'abord) :
+- Démarrer imerge merge (se placer dans la branche depuis laquelle "merger" d'abord) :
```

Comment:
Keep "frenglish" everywhere?
```suggestion
- Démarrer imerge merge (se placer dans la branche depuis laquelle merger d'abord) :
```

Reply:
I'm sorry, I don't quite get what you mean.  Are you referring to the fact that `merge` isn't a French word?

Reply:
Yes, the word `fusionner` is the equivalent French verb (it is used in [French Git Documentation](https://git-scm.com/docs/git-merge/fr)). But many French-speaking developpers don't use it because of the `git merge` command. :wink:

Reply:
Oh, well then if it's used in the official French Git docs, I'd say definitely use it here 👍🏻

Reply:
OK, I may do it in an other pull request... 😇